qgcs-config/product.json

```diff
@@ -8,6 +8,11 @@
         "read": "qgcs.utm_adapter.read",
         "write": "qgcs.utm_adapter.write"
     },
+    "utmsp_oauth2_data": {
+        "token_url": "https://utmsp.com/oauth/token",
+        "client_id": "client_id",
```

Comment:
The Client ID and especially the Client Secret cannot be in the `product.json` file, we can discuss this in this week's call and finalize